src/sqlfluff/core/parser/segments/base.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         name: Optional[str] = None,
     ):
         # A cache variable for expandable
-        self._is_expandable = None
```

Comment:
I added type hints to several existing properties in this class...

Reply:
Nit, but in future could you comment on the line below?

GitHub shows a few lines above, but none below so when you comment on the line above, it's difficult to see the full context in the Conversation tab. Noticed it on another PR too so thought I'd badger you 😉